core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt

```diff
@@ -83,6 +83,7 @@ import kotlinx.serialization.builtins.serializer
 import mu.KotlinLogging
 
 val logger = KotlinLogging.logger {}
+val SPEED_LIMIT_TAGS_RESOURCE_PATH = "speed_limit_tags.yml"
```

Comment:
I don't know if we want this to be available in the package (for now only one class in it, later on maybe more) and all the packages using `kt-osrd-rjs-parser`. Maybe change this to:
`private const val SPEE...`?

Reply:
Done, thanks (amended fixup commit): https://github.com/OpenRailAssociation/osrd/compare/8c7694d388f024f624de3db69c97ce03a8527567..b9073205c73c4dbf5a8573c8fef01d08accb99a0